src/components/send/components/AmountInput.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import styled from 'styled-components'
+
+const StyledInput = styled.input`
+    text-align: center;
+    font-weight: 600;
+    padding: 0;
+    height: 73px;
+    border: 0;
+
+    :focus {
+        border: 0;
+        box-shadow: none;
+    }
+
+    ::placeholder {
+        color: #CCCCCC;
+    }
+
+    &.error {
+        color: #FC5B5B;
+    }
+`
+
+const getFontSize = (charLength) => {
+    const baseSize = 60
+    if (charLength >= baseSize) {
+        charLength = baseSize - 5
+    }
+    const fontSize = baseSize - charLength
+    return fontSize
+}
+
+const AmountInput = ({ value, onChange, error }) => {
```

Comment:
Yep the idea was to keep this as pure UI as possible we can either wrap it in a new component or provide it with props. Similar to how the current `SendContainer.js` is handling amount validation.

> Should we have any formatting behavior here for decimals?

Curious what type of decimal formatting behavior you'd prefer? My POV is cleaner/less is more in this scenario of inputting an amount.

> Should we make the default value be a float including a decimal?

It might be a good idea, current is just the [default design](https://www.figma.com/file/unSbZpLnFv0oYBcU1gURWR/Transfer-Fungible-Tokens?node-id=415%3A291) but can tweak it before we ship.

> Assuming we'll be using this to send NFTs:
> 
> - Should there be validation here so that we can force people to only provide integer values in some cases?
> - Or should we perhaps create an entirely separate input for sending NFTs that disallows entering a float value?

I believe we'd want a separate flow for sending NFTs since you'd be sending a 'collectible' and may benefit from seeing the 'picture' vs a number.

Reply:
:+1: Seems like a new component for the transfer of NFTs is GTG :+1: